src/test/resources/andreeaG/suita Mobile tema8.xml

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="Mobile" verbose="1" >
+    <test name="mobile testing" >
+        <groups>
+            <run>
+                <include name="mobile"/>
+            </run>
+        </groups>
+        <classes>
+            <class name="andreeaG.tests.tema8.pageTests.BaseTest" />
```

Comment:
Please remove BaseTest (there are no tests to be run from this class) from this file and add DisabledDynamicButtonsTest instead of it. Thanks!